lib/mix_unused_discoveries.ex

```diff
@@ -10,6 +10,7 @@ defmodule MixUnusedDiscoveries do
     MixUnusedDiscoveries.Scout.Amqpx,
     MixUnusedDiscoveries.Scout.Application,
     MixUnusedDiscoveries.Scout.Exq,
+    MixUnusedDiscoveries.Scout.ExqMiddleware,
```

Comment:
Rinominerei quindi anche `MixUnusedDiscoveries.Scout.Exq` in `MixUnusedDiscoveries.Scout.ExqEnqueue`, o qualcosa di simile.